cscope.files

```diff
@@ -0,0 +1,810 @@
+./hollow-zenoadapter/src/test/java/com/netflix/hollow/zenoadapter/util/ObjectIdentityOrdinalMapTest.java
```

Comment:
remove this file and the other cscope file from the diff

Reply:
done